### ISSUE TITLE
Multiple pool address support

### ISSUE THREAD

**Issue by auc0le** [jstefanop/apolloapi-v2#8]
Are there plans to add multiple pool address support as the original software supported?  This is handy when one address / gateway goes down as the node can roll over to another geographic gateway or pool.

Thanks

**Comment by SearchAndDestroy** [comment]
Yes, please add multipool support so it can switch in case the 1st is  down or even switch to solo mining if the local ckpool were working

**Comment by Sledge0001** [comment]
Having a failover pool would indeed be ideal.
